content/news/2013-06-03-lightning-talks.md

```diff
@@ -4,6 +4,7 @@ Event: 2013-06-11 7:30 pm to 9:00 pm
 Location: O'Reilly Media
 Date: 2013-06-03 11:07
 Drupal_Node: 192
+Slug: lightning-talk-2013
```

Comment:
Or maybe I am wrong.  You got warnings from the build about this?  Just make sure that the actual HTML is generated at `/node/192/index.html` and we should be okay.